paddle/fluid/framework/details/build_strategy.cc

```diff
@@ -60,29 +76,44 @@ class ParallelExecutorPassBuilder : public ir::PassBuilder {
 
     // Add automatically inplace.
     if (strategy_.enable_inplace_) {
+      VLOG(10) << "Add inplace_pass";
       AppendPass("inplace_pass");
     }
 
+    if (strategy.fuse_elewise_add_act_ops_) {
+      VLOG(10) << "Add fuse_elewise_add_act_pass";
+      AppendPass("fuse_elewise_add_act_pass");
+    }
+
+    // for single card training, fuse_all_reduce_ops is unnecessary.
+    if (strategy.fuse_all_reduce_ops_) {
```

Comment:
doesn't need build_strategy? Can be calculated automatically?

Reply:
I think we should have `strategy.fuse_all_reduce_ops_` currently, and the default value can be set True.